services/web/server/src/simcore_service_webserver/folders/_models.py

```diff
@@ -36,35 +40,14 @@ class FolderFilters(Filters):
     )
 
 
-class FolderListSortParams(BaseModel):
-    # pylint: disable=unsubscriptable-object
-    order_by: Json[OrderBy] = Field(
-        default=OrderBy(field=IDStr("modified"), direction=OrderDirection.DESC),
-        description="Order by field (modified_at|name|description) and direction (asc|desc). The default sorting order is ascending.",
-        example='{"field": "name", "direction": "desc"}',
-        alias="order_by",
-    )
-
-    @validator("order_by", check_fields=False)
-    @classmethod
-    def _validate_order_by_field(cls, v):
-        if v.field not in {
-            "modified_at",
-            "name",
-            "description",
-        }:
-            msg = f"We do not support ordering by provided field {v.field}"
-            raise ValueError(msg)
-        if v.field == "modified_at":
-            v.field = "modified"
-        return v
-
-    class Config:
-        extra = Extra.forbid
+_FolderOrderQueryParams: type[RequestParameters] = create_ordering_query_model_classes(
+    ordering_fields={"modified", "name", "description"},
+    default=OrderBy(field=IDStr("modified"), direction=OrderDirection.DESC),
+)
 
 
-class FolderListWithJsonStrQueryParams(
-    PageQueryParameters, FolderListSortParams, FiltersQueryParameters[FolderFilters]
+class FoldersListQueryParams(
+    PageQueryParameters, _FolderOrderQueryParams, FiltersQueryParameters[FolderFilters]  # type: ignore[misc, valid-type]
 ):
     folder_id: FolderID | None = Field(
```

Comment:
folder_id and workspace_id here can also be None (more specifically, I think the frontend sends it as a "null" string). This option is not reflected in the OpenAPI specs. NOTE: This was likely the case previously as well.

Reply:
But this is already in place, right?

Reply:
clarified offline.